etc/kernel-launchers/R/scripts/launch_IRkernel.R

```diff
@@ -223,6 +223,10 @@ if (!file.exists(connection_file)){
     }
 }
 
+if (!is.na(argv$customAppName)){
+    list.append(sparkConfigList, spark.app.name=argv$customAppName)
```

Comment:
Reading up on things, it looks like `list.append` requires an `rlist` package.  The following might work instead of `list.append()`:
```R
sparkConfigList[['spark.app.name']] <- argv$customAppName
```